insights/configtree/__init__.py

```diff
@@ -0,0 +1,721 @@
+"""
+configtree
+==========
+configtree models configurations that are similar to dictionaries except that
```

Comment:
Please add this module to `docs/api_index.rst` so that it will be included in the built documentation.  Any docstrings you could add to public functions without docstrings would be helpful.